packages/server/src/__tests__/ApolloServer.test.ts

```diff
@@ -141,6 +141,40 @@ describe('ApolloServer construction', () => {
     // @ts-expect-error
     takesConfig({ modules: [] });
   });
+  describe('with configuration options', () => {
+    it('test', async () => {
```

Comment:
```suggestion
    it('stringifyResult', async () => {
```

Reply:
*sigh* - sorry this was a dumb one. Rushing through it!